README

```diff
@@ -28,12 +28,17 @@ When submitting questions and problems, be sure to include as much
 extra information as possible.  This web page details all the
 information that we request in order to provide assistance:
 
-     http://pmix.github.io/pmix/community/help/
+     https://pmix.org/support/
 
 The best way to report bugs, send comments, or ask questions is to
-sign up on the PMIx mailing list, which is hosted by GoogleGroups:
+post them on the OpenPMIX GitHub issue tracker:
```

Comment:
Putting on my Jeff Squyres "nit-picker" hat: it should be "OpenPMIx" - the "x" is not capitalized.

Reply:
Fair point! ;-)

Reply:
Fixed.